packages/now-build-utils/src/prerender.ts

```diff
@@ -7,21 +7,26 @@ interface PrerenderOptions {
   expiration: number;
   lambda: Lambda;
   fallback: FileBlob | FileFsRef | FileRef;
-  group: number;
+  group?: number;
 }
 
 export class Prerender {
   public type: 'Prerender';
   public expiration: number;
   public lambda: Lambda;
   public fallback: FileBlob | FileFsRef | FileRef;
-  public group: number;
+  public group?: number;
 
   constructor({ expiration, lambda, fallback, group }: PrerenderOptions) {
     this.type = 'Prerender';
     this.expiration = expiration;
     this.lambda = lambda;
     this.fallback = fallback;
+    
+    if (group && group <= 0 || !Number.isInteger(group)) {
```

Comment:
```suggestion
    if (!Number.isInteger(group as number)) {
```

For some reason, TS thinks that you can only pass a number to `Number.isInteger()` 🤔 

Related https://github.com/microsoft/TypeScript/issues/21199

Reply:
See correct behavior here: https://github.com/zeit/now/pull/3082#discussion_r328813500